Library/Homebrew/keg_relocate.rb

```diff
@@ -1,4 +1,12 @@
+require "os/mac/relocate"
```

Comment:
If possible, I would like to have `require "os/mac/..."` only exist inside `os` folder to have a better OS abstraction. So I suggest to create an `os/mac/keg.rb` file similar to `os/mac/pathname.rb` file.